### ISSUE TITLE
Test new beacon baselines in simulation (Trac #250)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#250]
It's probably as simple as hacking DOMsimulator to use the beacon baselines in the DB.  This would allow for a uniform treatment of simulation and data.

Migrated from https://code.icecube.wisc.edu/ticket/250
```json
{
    "status": "closed", 
    "changetime": "2019-01-11T22:45:06", 
    "description": "It's probably as simple as hacking DOMsimulator to use the beacon baselines in the DB.  This would allow for a uniform treatment of simulation and data.", 
    "reporter": "olivas", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1547246706829407", 
    "component": "combo simulation", 
    "summary": "Test new beacon baselines in simulation", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2011-04-14T04:00:00", 
    "milestone": "", 
    "owner": "olivas", 
    "type": "enhancement"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20111104T18:25:15: 
* olivas changed resolution from "" to "invalid"
* olivas changed status from "new" to "closed"

**Comment by icecube-trac** [comment]
Trac update at 20190111T22:45:06: 
* nwhitehorn commented:

> In changeset 169661:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="169661"
> closes #250
> ```


* nwhitehorn changed resolution from "invalid" to "fixed"
